content/AddedListPublicHealthInterventions.introduction.md

```diff
@@ -108,6 +108,7 @@ These can include characteristics such as when infected individuals are contagio
 The development of diagnostic tools is critical to this goal.
 Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
 Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
+Possible public health intervention types for prevention and control in general include: diagnostics, data gathering, pharmaceutical treatment, pharmaceutical prophylaxis, vaccines, and non-pharmaceutical measures such as social distancing, hand hygiene, cough/sneeze hygiene or practices, contact tracing, isolation, for particularly severe pandemics sometimes the use of different types of facemasks as useful and sometimes measures for reduction of contacts, and personal protective equipment for health workers [@ISBN:978-92-4-156553-0].
```

Comment:
```suggestion
Beyond epidemiology, the field of public health also encompasses data gathering related to the impact of the pandemic on society and on subsections of society, as well as interventions targeted at prevention and control.
These interventions can include efforts to educate the public about pharmaceutical advances such as diagnostics, treatment, and prophylaxis including vaccines, as well as non-pharmaceutical measures such as social distancing, hand hygiene, cough/sneeze hygiene or practices, and personal protective equipment, e.g., different types of face masks [@ISBN:978-92-4-156553-0].
One public health strategy for management of the pandemic that has attracted particular attention is the use of test, trace, and isolate methods for containment [@url:https://healthpolicy.duke.edu/sites/default/files/atoms/files/covid-19_surveillance_roadmap_final.pdf].
```
@yemarshall I am not sure if this is helpful, but based on my conversation with @juliettemarie0405 and another MPH from a global health background, [Ipsita Krishnan](https://www.linkedin.com/in/ipsitakrishnan/), I tried to break some of your ideas out and to sign-post what broader ideas they link into! I don't know what you'll think though!

Reply:
(just a note, though this isn't the last version suggested -- not sure if we should keep or skip the mention of PPE / "personal protective equipment" & facemask, but if we do keep these we may need to separate PPE from the more variable term of facemask / facial covering. For PPE for healthworkers I mentioned in part b/c they work in a crucial way with symptomatic people in complicated ways; meanwhile, the facemask or face covering terms have been more variable, maybe that's why it's easier to just skip this as is in the last version suggested. But I'm not sure especially b/c 1) SARS & MERS research mentions nosocomial transmissions [still under research which transmission events major for SARS-CoV-2 as I added elsewhere but of course important to prevent this type of transmission regardless] so makes sense to have PPE in addition to the general PPE recommendations for infectious diseases & 2) different facemasks or facial coverings have been suggested though not always required and also facemasks for sick people are mentioned in the manual for especially strong epidemics of influenza pandemic. I will think about it again.)

Reply:
comment I put above was not written very clearly, sorry. I'll think about it again & right more clearly later. maybe if we include PPE then worth just saying PPE & skipping the more complicated thing about different types and reasons for facemasks/facial coverings.

Reply:
@yemarshall please feel free to ignore this one :) I am an evolutionary biologist but since so many people in my life are in public health, so I just asked them for advice and tried to paraphrase it! I'd recommend following @juliettemarie0405 below rather than this one!